src/state/quick-bar-mixin.ts

```diff
@@ -43,6 +48,29 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
       showQuickBar(this, { commandMode });
     }
 
+    private _createMyLink(e: KeyboardEvent) {
+      if (!this._canOverrideAlphanumericInput(e) || !this.hass) {
+        return;
+      }
+      const targetPath = mainWindow.location.pathname;
+
+      for (const [path, redirect] of Object.entries(
+        getMyRedirects(isComponentLoaded(this.hass, "hassio"))
+      )) {
+        if (redirect.redirect === targetPath) {
+          window.open(
+            `https://my.home-assistant.io/create-link/?redirect=${path}`,
+            "_blank"
+          );
+
+          return;
+        }
+      }
+      showToast(this, {
+        message: `No matching my link found for ${targetPath}`,
```

Comment:
Translation would be great 😄

Reply:
I'll add that if there is a sign of it being accepted.

Reply:
I like the idea.